third_party/blink/renderer/core/farbling/brave_session_cache.h

```diff
@@ -35,6 +51,17 @@ GetBraveFarblingLevelFor(ExecutionContext* context,
 CORE_EXPORT bool AllowFingerprinting(ExecutionContext* context);
 CORE_EXPORT bool AllowFontFamily(ExecutionContext* context,
                                  const AtomicString& family_name);
+CORE_EXPORT int MaybeFarbleScreenInteger(ExecutionContext* context,
+                                         brave::FarbleKey key,
+                                         int spoof_value,
+                                         int min_value,
+                                         int max_value,
+                                         int defaultValue);
+CORE_EXPORT bool BlockScreenFingerprinting(ExecutionContext* context);
+CORE_EXPORT int FarbledPointerScreenCoordinate(const DOMWindow* view,
+                                               FarbleKey key,
+                                               int clientCoordinate,
+                                               int trueScreenCoordinate);
```

Comment:
pls check code style across the PR: `trueScreenCoordinate`

Reply:
Thanks -- I fixed such style things across the PR.